draft-ietf-intarea-provisioning-domains.md

```diff
@@ -547,7 +545,7 @@ is not defined in this document.
 When the H-flag of the PvD Option is set, hosts MAY attempt to
 retrieve the PvD Additional Information associated with a given PvD by
 performing an HTTP over TLS {{?RFC2818}} GET query to
-https://\<PvD-ID\>/.well-known/pvd {{!RFC8615}}.
+https://\<PvD-ID\>/.well-known/pvd.
```

Comment:
Humm I do not agree with Suresh's question about the presence of RFC 8615. For me, this is relevant and should be a normative reference.
Up to you Tommy ;-)

Reply:
Well, I just moved it down to where we add the IANA registration, since it is definitely relevant there.